src/main/java/com/google/javascript/clutz/AliasMapBuilder.java

```diff
@@ -0,0 +1,67 @@
+package com.google.javascript.clutz;
+
+import com.google.javascript.rhino.Node;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * AliasMapBuilder builds a map from the imported name of a symbol (based on a goog.require) to the
+ * exported name of a symbol (based on the variable name and goog.module id), so symbols that are
+ * imported and reexported can be properly aliased.

```

Comment:
needs an example, I can't fully follow the text without an example here.

Word it like, given this code:
const A = goog.require('x.y.z') the map will contains A -> x.y.z,

doesnt' need to cover all cases.

Reply:
Reworked the comment - thoughts?